File: python/rna-transcription/rna_transcription.py
def to_rna(dna_strand):
    dna_to_rna={'G':'C','C':'G','T':'A','A':'U',' ':''}
    rna=""
    for nucleotide in dna_strand:
        rna+=(dna_to_rna[(nucleotide)])
    return rna
    pass

File: javascript/rna-transcription/rna-transcription.js
//
// This is only a SKELETON file for the 'RNA Transcription' exercise. It's been provided as a
// convenience to get you started writing code faster.
//
const STRANDS={G:'C',C:'G',T:'A',A:'U'}
export const toRna = (DNA) => {
  return Array.from(DNA).map(item=>STRANDS[item]).join('')
};
